packages/mantine/src/components/header/Header.tsx

```diff
@@ -48,7 +48,11 @@ export const Header: HeaderType = ({description, borderBottom, children, variant
             >
                 <Stack spacing={0}>
                     {breadcrumbs}
-                    <Title order={variant === 'page' ? 1 : 3} color={variant === 'page' ? 'gray.5' : undefined}>
+                    <Title
+                        order={variant === 'page' ? 1 : 3}
+                        color={variant === 'page' ? 'gray.5' : undefined}
+                        style={{wordBreak: 'break-all'}}
```

Comment:
```suggestion
                        sx={{ wordBreak: "break-word" }}
```

I think `break-word` would be more appropriate than `break-all`. It would avoid breaking actual words in two whenever we have the space for it while still breaking words that are too long.

Break all
![image](https://github.com/coveo/plasma/assets/35579930/9a0824d0-5e82-4ed0-b7a1-171f37876b1e)

Break word
![image](https://github.com/coveo/plasma/assets/35579930/76e134df-b148-401a-886b-a45cd8410820)

Reply:
I see!! I will make the changes